gems/pending/VMwareWebService/MiqVimVdlMod.rb

```diff
@@ -33,20 +33,23 @@ module MiqVimVdlVcConnectionMod
 	# The connection is specific to this VM, and should be closed by the caller when it
 	# has finished accessing the VM's disk files.
 	#
-	def vdlVcConnection
+	def vdlVcConnection(thumb_print)
 	    require 'VixDiskLib'
 	
         VixDiskLib.init(lambda { |s| $vim_log.info  "VMware(VixDiskLib): #{s}" },
                         lambda { |s| $vim_log.warn  "VMware(VixDiskLib): #{s}" },
                         lambda { |s| $vim_log.error "VMware(VixDiskLib): #{s}" })
 	    
-		$vim_log.info "MiqVimVmMod.vdlVcConnection: #{invObj.server}"
-	    VixDiskLib.connect(:serverName => invObj.server,
-	                       :vmxSpec    => vixVmxSpec,
-                           :port       => 902,
-                           :credType   => VixDiskLib_raw::VIXDISKLIB_CRED_UID,
-                           :userName   => invObj.username,
-                           :password   => invObj.password)
-    end
+		$log.info "MiqVimVdlVcConnectionMod.vdlVcConnection: server - #{invObj.server}"
+         thumb_print.to_cert
```

Comment:
There are many methods here, all of which seem to do things, rather than returning a result -- and none of which seem to be named with verbs.

(Incidentally, most of them sound like they *should* be queries, not commands... but if they're going to be commands, they at least need to be named like it.)